scripts/build-go-binaries.sh

```diff
@@ -10,17 +10,15 @@ do
 	platform_split=(${platform//\// })
 	GOOS=${platform_split[0]}
 	GOARCH=${platform_split[1]}
-	PACKAGE_NAME="github.com/NethermindEth/nethermindEth/sedge"
 	SEDGE_VERSION=$(git tag | sort | tail -n 1)
-	output_name=$package_name'-v'$VERSION'-'$GOOS'-'$GOARCH
+	output_name=$package_name'-'$SEDGE_VERSION'-'$GOOS'-'$GOARCH
```

Comment:
Does SEDGE_VERSION has the `v` prefix? If not please add it here

Reply:
yes, SEDGE_VERSION basically puts the tag, and we include the `v` on the tags, for example `v0.3.0`